### PR TITLE
Added Emerging Threats Block List for lookups

### PR DESCRIPTION
**Pull request by kurobeats**
I'm no java dev, but gave it a crack. Seems to be functioning in my tests.

### CODE REVIEW

File: src/main/java/org/graylog/plugins/threatintel/ThreatIntelPluginModule.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import org.graylog.plugins.threatintel.providers.otx.domain.OTXDomainLookupFunction;
 import org.graylog.plugins.threatintel.providers.otx.ip.OTXIPLookupFunction;
 import org.graylog.plugins.threatintel.providers.spamhaus.SpamhausIpLookupFunction;
+import org.graylog.plugins.threatintel.providers.emerging_threatsIpLookupFunction;
```
Comment (lennartkoopmann) [inline src/main/java/org/graylog/plugins/threatintel/ThreatIntelPluginModule.java]: This is causing a compilation failure.

File: src/main/java/org/graylog/plugins/threatintel/providers/emerging_threats/emerging_threatsIpLookupFunction.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import org.graylog.plugins.pipelineprocessor.ast.functions.AbstractFunction;
+import org.graylog.plugins.pipelineprocessor.ast.functions.FunctionArgs;
+import org.graylog.plugins.pipelineprocessor.ast.functions.FunctionDescriptor;
+import org.graylog.plugins.pipelineprocessor.ast.functions.ParameterDescriptor;
+import org.graylog.plugins.threatintel.providers.GenericLookupResult;
+import org.graylog2.plugin.cluster.ClusterConfigService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class emerging_threatsIpLookupFunction extends AbstractFunction<GenericLookupResult> {
```
Comment (lennartkoopmann) [inline src/main/java/org/graylog/plugins/threatintel/providers/emerging_threats/emerging_threatsIpLookupFunction.java]: Please rename to `EmergingThreatsIpLookupFunction`

File: src/main/java/org/graylog/plugins/threatintel/providers/emerging_threats/emerging_threatsIpLookupFunction.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            LOG.error("Could not run emerging_threats lookup for IP [{}].", ip, e);
+            return null;
+        }
+    }
+
+    @Override
+    public FunctionDescriptor<GenericLookupResult> descriptor() {
+        return FunctionDescriptor.<GenericLookupResult>builder()
+                .name(NAME)
+                .description("Match an IP address against the emerging_threats DROP and EDROP lists.")
```
Comment (lennartkoopmann) [inline src/main/java/org/graylog/plugins/threatintel/providers/emerging_threats/emerging_threatsIpLookupFunction.java]: `DROP` and `EDROP` is from Spamhaus and not from Emerging Threats

**Comment by dennisoelkers** [comment]
This PR is totally out of date, remarks had not been addressed and in the current version you can just use a lookup table without any code changes, so I am closing this PR.

**Pull request closed without merging**